### DISCUSSION TITLE
请教下authentication流程中那些url以及参数是从哪里获取的？

### DISCUSSION THREAD

**Discussion by CountOfGrubby** [InfpHub/Raptor#10] [General]
请教下authentication流程中那些url以及参数是从哪里获取的？
我翻遍了文档也只有ios以及android的sdk
没有看到相关的http api呀

**Comment by InfpHub** [comment]
url 参数指的是 fromSuite＝52，类似这些参数吗🤔？如果是，这个参数都是官方网页版 API 请求的参数，通过网页版抓包得到的。

**Reply by CountOfGrubby** [reply]
谢谢谢谢。昨天一直在找文档，一直对应不上，上午看了下网页端的请求流程发现是一样的。
这个感觉工作量好大呀，网页端一改的话，整个流程都要重新抓取一遍。
